src/test/datascience/plotViewer/plotViewer.vscode.test.ts

```diff
@@ -69,7 +70,12 @@ plt.show()`,
         await waitForExecutionCompletedSuccessfully(plotCell);
 
         await waitForCondition(async () => plotCell?.outputs.length >= 1, 10000, 'Plot output not generated');
-        assert(plotCell.outputs.length === 1, 'Plot cell output incorrect count');
```

Comment:
Changed this to use `assert.equal`, this way, we know what the output is, else we don't know whether the output is 0, 2, 20.